power_grid_model_c/power_grid_model/include/power_grid_model/math_solver/iterative_linear_se_solver.hpp

```diff
@@ -792,7 +866,7 @@ template <bool sym> class IterativeLinearSESolver {
                             rhs_block.eta() +=
```

Comment:
Same here
 But the order does not matter. Maybe at list specify the formula in the comment in the correct order.